.coderabbit.yaml

```
@@ -0,0 +1,96 @@
+chat:
+  auto_reply: true  # Enable auto-replies in PR comments
+
+# Verification settings
+verifications:
+  # Enable verification of fixes
+  verify_fixes: true
+  # Add custom verification steps
+  steps:
+    - name: "Verify TypeScript Compilation"
+      run: "tsc --noEmit"
+      on_failure: "TypeScript compilation failed. Please fix the type errors." 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)